app/models/concerns/contact/transferable.rb

```diff
@@ -16,6 +16,7 @@ def transfer(new_registrar)
     new_contact.regenerate_code
     new_contact.regenerate_auth_info
     new_contact.remove_address unless self.class.address_processing?
+    new_contact.uuid = nil
```

Comment:
Why is this required? Is there a reason why UUIDs cannot stay the same after a transfer, like primary keys?

Reply:
Because transfer operation effectively creates new contact, based on the original one.

Reply:
Folding issues, didn't see the `self.dup` at the beginning. All good 👍 

<img width="790" alt="screen shot 2018-06-13 at 12 29 55" src="https://user-images.githubusercontent.com/2331765/41342691-868329c8-6f05-11e8-94f8-66513624e8f4.png">